src/ImageSharp/Processing/Processors/Dithering/ErrorDiffusionPaletteProcessor{TPixel}.cs

```diff
@@ -49,7 +47,7 @@ protected override void OnFrameApply(ImageFrame<TPixel> source)
             sourcePixel.ToRgba32(ref rgba);
 
             // Convert to grayscale using ITU-R Recommendation BT.709 if required
-            byte luminance = isAlphaOnly ? rgba.A : ImageMaths.Get8BitBT709Luminance(rgba.R, rgba.G, rgba.B);
```

Comment:
Dropped this check because is was a hack.`Alpha8` should never work with error diffusion as the luminance is a constant..